third_party/terraform/utils/image_test.go

```diff
@@ -20,9 +20,8 @@ func TestAccComputeImage_resolveImage(t *testing.T) {
 	fam := fmt.Sprintf("test-image-family-%s", rand)
 
 	resource.Test(t, resource.TestCase{
-		PreCheck:     func() { testAccPreCheck(t) },
-		Providers:    testAccProviders,
-		CheckDestroy: testAccCheckComputeImageDestroy,
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
```

Comment:
I think the CheckDestroy should still be in here.

Also while you're here, I think this test should be moved into resource_compute_image_test.go so we can keep all the hand-written acceptance tests in the same file.

Reply:
Done!  In the meantime, fixed checkdestroy's tendency to check whether we had destroyed data sources with the same type name as the resource in question.  :)